app/Response/Export.hs

```diff
@@ -8,14 +8,7 @@ import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
 import Happstack.Server
 
+
 exportResponse :: ServerPart Response
 exportResponse =
-    ok $ toResponse $
-        displayPDF
-
-
-{- Insert a large loading icon into the page -}
-displayPDF :: H.Html
-displayPDF =
-    H.div ! A.id "exportDiv" $
-        H.img ! A.id "export-pdf" ! A.src "../CSgraph.pdf"
+    serveDirectory DisableBrowsing ["CSGraph.pdf"] "."
```

Comment:
Better, although it seems like there should be a `serveFile` function to use instead